Detectors/AOD/src/AODProducerWorkflowSpec.cxx

```diff
@@ -846,9 +966,14 @@ void AODProducerWorkflowDPL::run(ProcessingContext& pc)
   uint32_t labelTPC;
   uint16_t labelMask;
 
+  // filling track labels
```

Comment:
I think we need more complete treatment of labels. E.g. ITS-TPC track label will be that of the TPC track, with the isFake()==true of the ITStrack label != TPCtrack label. Don't we want to store the labels of these contributors? This is the only way to know if e.g. ITS or TPC track is fake (i.e. has wrong clusters attached).